### PR TITLE
LPS-142436 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-142436

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUseCase.testcase [hunk 1]
```diff
@@ -27,7 +27,7 @@ definition {
 		else {
 			User.logoutAndLoginPG(userLoginEmailAddress = "test@liferay.com");
 
-			Role.tearDownCP();
+			JSONRole.deleteRole(roleTitle = "Regular Role Name");
```
Comment (ozysouza) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUseCase.testcase:30]: Hi @Tim-Cao, after rethink I'll remove this change, it's going to break the tests that doens't has Regular Role, but I also will fix the Role.tearDownCP(); it's not working, there is a conditional that doens't let the role be deleted.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdmin.testcase [hunk 2]
```diff
@@ -293,562 +218,239 @@ definition {
...
-		PageEditor.addFragment(
-			collectionName = "Content Display",
-			fragmentName = "Display Page Content");
-
-		PageEditor.clickPublish();
-
-		for (var i : list "1,2,3") {
+	@description = "This is a test for LPS-110183. Can access to edition of collection items in Collection Items modal."
+	@priority = "4"
+	test EditAssetsViaCollectionItems {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdmin.testcase:269]: GoToCollectionItemEditModeViaViewCollectionItemsModal

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdmin.testcase [hunk 3]
```diff
@@ -1229,251 +835,463 @@ definition {
...
+
+			for (var pageName : list "Page One,Page Two") {
+				AssertElementNotPresent(locator1 = "//tr/td[2][contains(.,'${pageName}')]");
+			}
+		}
+	}
+
+	@description = "This is a test for LPS-110183. Can view information about collection items of collection page via Pages admin."
+	@priority = "5"
+	test ViewAssetsInCollectionItem {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdmin.testcase:896]: Keep the original name ViewCollectionItemViaPageFinderAndPageAdmin

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update according to comments? Btw, please resolve rebase conflict. Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update according to comments? Btw, please resolve rebase conflict. Thanks

Hi @Tim-Cao, PR updated following your comments and conflicts solved. Would you mind run the SF locally to see if everything is okay? I don't know why but my source format is not working today, I've let a wrong formatation on porpuse to check and the build got successeful even though it should fail then I did the SF manually and I maybe missed something. Thanks you

**Comment by Tim-Cao** [comment]
ci:test:sf

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update according to new comments? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesFinder.testcase [hunk 4]
```diff
@@ -268,6 +273,141 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-129406. The user that has all permissions for Web Content in Content & Data role can not add a page via Page Tree Panel."
+	@priority = "4"
+	test CannotAddPageWithAllWebContentPermissions {
+		task ("Add a new user") {
```
[the remark this answers is not in the document]
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesFinder.testcase:279]: Hi @Tim-Cao, I still didn't understand very well when should I use this property, could you explaing it for me?

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update according to new comments? Thanks!

Hi @Tim-Cao, PR updated with the last changes.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/8e1cd277d61039926a00937be302be3e50a9ba75...56b59f5e2db6e85da46bf034d4f94fc148c989fd

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**